package.json

```diff
@@ -4,11 +4,11 @@
   "description": "Collection of devtools packages",
   "scripts": {
     "flow": "flow",
-    "lint": "yarn run lint-css -s; yarn run lint-js -s",
+    "lint": "npm run lint-css -s; a=$?; npm run lint-js -s; b=$?; test $a -eq 0 -a $b -eq 0",
```

Comment:
We could have used a simpler `npm run lint-css && npm run lint-js` but I wanted to always run both linters even if lint-css was failing, as before. But this would also work for me, so up to you.